CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## [Unreleased]
 
+### Added
+
+- support for Python 3.12
+- support for Pylint 3
+
+### Removed
+
+- support for Python 3.6 & 3.7
```

Comment:
Please:

- Sentence case for the entries
- Link the relevant tickets

Ik there's no linter for that 🙏 sorry for the legwork